source/_data/documentation.yml

```diff
@@ -0,0 +1,77 @@
+toc:
+  - Syntax: /documentation/syntax/
+    :children:
```

Comment:
Why do we need `:children:`?

Reply:
Because each section is an object, we can't just embed a list of children without a key (in this case, `:children`). I considered reformatting this in a more verbose way, but decided to stick with the existing structure since it's succinct and what they're used to.